packages/protocol/contracts-0.8/governance/Validators.sol

```diff
@@ -713,17 +713,23 @@ contract Validators is
    * @param account The address of the validator group.
    * @param n The number of members to return.
    * @return The top n group members for a particular group.
+   * @dev Returns the account instead of signer on L2.
    */
   function getTopGroupValidators(
```

Comment:
So this changes the behaviour of this function, to try to avoid that, can we add a new function that's called something like `getTopGroupValidatorsAccounts` that works in L1/L and returns just topAccounts. And then add some sort of `getTopGroupValidatorsL2` that returns `getTopGroupValidators ` on L1 but `getTopGroupValidatorsAccounts` on L2?

Then election should be updated to call `getTopGroupValidatorsL2` instead of `getTopGroupValidators`.

I know there's tooling for people watching if their validator will soon be elected that would break if we change this function.

Reply:
Not sure I follow. The concept of signers is deprecated on L2. Wouldn't the tooling break once migrated to L2 regardless? 

Any tooling they use should be modified to use account instead of signer, no?

Reply:
left `getTopGroupValidators()` as is, and created `electValidators()` that replaces `electValidatorSigners` on L2.

This means that `electValidatorSigners` can only be called on L1 and `electValidators()` can only be called on L2

Reply:
The concept of singers is not deprecated in L2, they will still be in the accounts, just that it wont be used.